README.md

```diff
@@ -5,18 +5,18 @@ This library provides a set of utilities for performing automatic detection of a
 **Compatible with Python 3.x only.**  Sorry.
 
 * **Install:** ``pip install wikiclass``
-* **Models:** https://github.com/wiki-ai/wikiclass/tree/master/models
+* **Models:** https://github.com/wiki-ai/articlequality/tree/master/models
 * **Documentation:** http://pythonhosted.org/wikiclass
```

Comment:
http://pythonhosted.org/articlequality (we'll need to upload docs soon after the merge)

Reply:
404s for me, that's why I thought we should do it in several steps :D